sqlglot/dialects/duckdb.py

```diff
@@ -41,6 +42,14 @@
     exp.DateAdd, exp.TimeAdd, exp.DatetimeAdd, exp.TsOrDsAdd, exp.DateSub, exp.DatetimeSub
 ]
 
+WINDOW_FUNCS = (
```

Comment:
can we make this a subclass instead? change the expression subclass to window func

Reply:
I think these are a subset of the actual window funcs, right @VaggelisD? IIRC duckdb only allows `IGNORE NULLS` for certain window funcs, hence this tuple? If so, I'd actually suggest renaming this to something like `WINDOW_FUNCS_WITH_IGNORE_NULLS` or something.

Reply:
Right, this was a detail that I should have documented more clearly; DuckDB differentiates the aggregate functions from the general-purpose window functions, and `IGNORE NULLS` is accepted on a subset of the general-purpose window funcs. 

This [table here](https://duckdb.org/docs/sql/functions/window_functions.html#general-purpose-window-functions) documents them, note those with `IGNORE NULLS`. The `WINDOW_FUNCS` set holds this subset, but it's not accurately named as George pointed out, will rename it.

Reply:
got it